src/fides/api/service/connectors/saas_connector.py

```diff
@@ -861,23 +972,31 @@ def _invoke_masking_request_override(
 
     @staticmethod
     def _invoke_consent_request_override(
+        override_function: RequestOverrideFunction,
         override_function_name: str,
```

Comment:
In my previous implementation, I passed in `override_function_name` because I performed the function lookup inside `_invoke_consent_request_override`. Now, we look up the function outside because we need it to determine which flow to use. I think in our new use case we can remove this param, and just call `override_function.__name__` if we want to use the function name in the logs. Better to pull the name directly from the function.

Reply:
good call!